tests/emails/reset_password_test.py

```diff
@@ -0,0 +1,56 @@
+from unittest.mock import patch
+
+from emails.user_reset_password import make_reset_password_email_data
+from tests.test_utils import create_user
+
+
+class MakeResetPasswordEmailDataTest:
+    @patch('emails.user_reset_password.SUPPORT_EMAIL_ADDRESS', 'support@example.com')
+    @patch('emails.user_reset_password.format_environment_for_email', return_value='-testing')
+    @patch('emails.user_reset_password.feature_send_mail_to_users_enabled', return_value=False)
+    def test_should_write_email_with_right_data_when_not_production_environment(self, mock_send_mail_enabled,
```

Comment:
`with_right_data` n'apporte aucune information

```suggestion
    def test_email_is_sent_to_dev_at_passculture_when_not_production_environment(self, mock_send_mail_enabled,
```